### PR TITLE
Make /dashboard an Ember app

### PR DESCRIPTION
**Pull request by locks**
Uses the same technique as /builds.

Preview: https://deploy-preview-2820--chess-master-boris-50065.netlify.com/dashboard/

### CODE REVIEW

**Comment by stefanpenner** [comment]
i like the loading indicator!

**Comment by locks** [comment]
Courtesy of the lovely [spinkit](http://tobiasahlin.com/spinkit/) collection :)

File: source/javascripts/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    var element;
+    var i;
+
+    for (i=0; i < tagInfo.length; i++) {
+      element = document.createElement('script');
+      element.src = bucketURL + tagInfo[i].src;
+      // element.integrity = tagInfo[i].integrity;
+      element.async = false;
+
+      console.log(tagInfo[i].src);
```
Comment (acorncom) [inline source/javascripts/dashboard.js]: @locks Looks like we need to take this out

**Comment by locks** [comment]
gtg

**Comment by rwjblue** [comment]
:shipit:

**Comment by acorncom** [comment]
🎉

**Pull request merged**